services/uploads/serverless.yml

```diff
@@ -119,6 +119,11 @@ resources:
                   s3:ExistingObjectTag/virusScanStatus:
                     - "CLEAN"
                   aws:PrincipalArn: !GetAtt BucketAVScanRole.Arn
+            - Action:
+                - "s3:PutObject"
+              Effect: "Deny"
+              Resource: !Sub ${AttachmentsBucket.Arn}/*.html
+              Principal: "*" 
```

Comment:
You're using a whitelist in the react code, but a black list in the bucket policy.  I recommend we keep it consistent; if a whitelist is what we've been given, let's use it in both places.

See docs on [how to restrict buckets to certain file types](https://aws.amazon.com/premiumsupport/knowledge-center/s3-allow-certain-file-types/)
That link above has an example of applying an extension whitelist to an S3 bucket's policy.  Pasting below....:
```
{
      "Sid": "Stmt1464968483619",
      "Effect": "Deny",
      "Principal": "*",
      "Action": "s3:PutObject",
      "NotResource": [
        "arn:aws:s3:::DOC-EXAMPLE-BUCKET/*.jpg",
        "arn:aws:s3:::DOC-EXAMPLE-BUCKET/*.png",
        "arn:aws:s3:::DOC-EXAMPLE-BUCKET/*.gif"
      ]
    }
```